connector_importer/components/importer.py

```diff
@@ -342,6 +350,7 @@ def run(self, record, is_last_importer=True, **kw):
                             continue
                         odoo_record = self.record_handler.odoo_create(values, line)
                         self.tracker.log_created(values, line, odoo_record)
+                    self._after_import(odoo_record, values, line)
```

Comment:
this can be done on the recordhandler no?

Reply:
yeah probably. I was expecting to use it later but that's not the case anymore. I'll remove it